layouts/partials/map.html

```diff
@@ -1,13 +1,13 @@
 {{ if isset .Site.Params "latitude" }}
   {{ if isset .Site.Params "longitude" }}
       <div class="hidden">
-        <input id="gmap-lat" value="{{ .Site.Params.latitude }}" />
-        <input id="gmap-lng" value="{{ .Site.Params.longitude }}" />
-        <input id="gmap-marker" value="{{ "img/marker.png" | relURL }}" />
+        <input type="hidden" id="gmap-lat" value="{{ .Site.Params.latitude }}" />
+        <input type="hidden" id="gmap-lng" value="{{ .Site.Params.longitude }}" />
+        <input type="hidden" id="gmap-marker" value="{{ .Site.BaseURL }}img/marker.png" />
```

Comment:
is this change needed ? {{ "img/marker.png" | relURL }}

Reply:
I'd strongly suggest to stick with `{{ "img/marker.png" | relURL }}` instead of `{{ .Site.BaseURL }}img/marker.png`. The latter results in an absolute URL which can cause portability issues (which one can [work around using Hugo's `--baseURL` option](https://discourse.gohugo.io/t/correct-baseurl-on-netlify-previews/10429); but if we can avoid the hassle from the beginning... :wink: )